src/site/content/en/media/prepare-media/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: Prepare media files for the web
+authors:
+  - joemedley
+description: |
+  Take a raw video file off a camera and transform it into an encrypted resource
+  that you can embed in a web page.
+date: 2017-06-30
+updated: 2020-08-20
+tags:
+  - FFmpeg
+  - files
+  - Shaka
+---
+
+Now that I've introduced you to [applications used for manipulating media
+files](../application-basics), over the next few pages, I'm going to take a raw
+video file off a camera and transform it into an encrypted resource that you can
+embed in a web page. I'm specifically going to show you how to format your video
+for mobile playback, and how to create multiple files to cover a range of
+browsers. Specifically, I'll create a webm file for use on Chrome and an mp4
+file for use on other browsers.
+
+{% Aside %}
+This section provides explanations of file manipulation concepts,
+with command lines only to illustrate the concepts. There is a companion [cheat
+sheet](../cheatsheet) that shows more commands and is designed as a quick reference
+for someone who knows the concepts.
+{% endAside %}
+
+The result of this procedure will be media resources with the following
+characteristics:
+
+
+* Versions of a media file in common web-friendly formats containing both audio
+  and video streams.
+* A resolution appropriate for your users' devices.
+* A bitrate that doesn't overload your users' network bandwidth.
+* (Optional) Encryption.
+* Viewable on all major browsers using appropriate technologies.
+
+By "appropriate technologies" I mean [Dynamic Adaptive Streaming over HTTP
+(DASH)](https://developer.mozilla.org/en-US/docs/Web/HTML/DASH_Adaptive_Streaming_for_HTML_5_Video)
+or [HTTP Live Streaming
+(HLS)](https://developer.apple.com/documentation/http_live_streaming), which are
+the two primary means of providing video in HTML on the major browsers. What
+those terms mean and how to use them is a whole topic itself. I won't be getting
+into those, but by the end of this article, you'll be able to create media files
+that are ready for use in DASH and HLS.
```

Comment:
We say we won't be getting into DASH and HLS, but it might be worth noting that they really don't need to know much about them.

Reply:
Added a suggestion to address this